vowpalwabbit/parse_example_json.h

```diff
@@ -1382,7 +1402,7 @@ void read_line_decision_service_json(vw& all, v_array<example*>& examples, char*
   json_parser<audit>* parser = static_cast<json_parser<audit>*>(all.p->jsonp.get());
 
   // As long as VW was configured correctly this should not occur.
-  if(parser->mode == json_parser_mode::standard)
+  if (parser->mode == json_parser_mode::standard)
```

Comment:
[Nit] `json_parser_mode::standard` seems to imply that we run parser in JSON "Standard" and JSON "non-standard" mode, where the real selection is between non-*CB parsing and *CB parsing.